index.html

```diff
@@ -8,29 +8,165 @@
+                  project you need coded,  don’t<br>
+                   hestiate to contact me.
+          </p>
+          <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+         </section>
+     </section>
+    <section class="skill">
+      <section class="languageframeworks">
+        <ul class="language">
+          <li id="language">Language</li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul class="language">
+          <li id="language">Frameworks</li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <img src="images/Ellipse 19.png" class="ellipse" alt="Ellipse">
+      </section>
+      <section class="skills">
+        <ul class="language">
+          <li id="language">Skills</li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlap</li>
+          <li>Tarminal</li>
+        </ul>
+      </section>
+    </section>
+    <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+    <ul class="socialmedia">
+      <li><a href="#"><img src="images/Vector.png" alt="Github_icon"></a></li>
+      <li><a href="#"><img src="images/Group.png" alt="Linkedin_icon"></a></li>
+      <li><a href="#"><img src="images/VectorX.png" alt="X_icon"></a></li>
+      <li><a href="#"><img src="images/Teiitter.png" alt="Twitter_icon"></a></li>
+      <li><a href="#"><img src="images/Vectorm.png" alt="M_icon"></a></li>
+    </ul>
+    <div class="scroll">
+      <img src="images/Group 55.png" class="arrow" alt="Arrow">
+      <img src="images/Work.png" class="work" alt="work">
+    </div>
+    <div class="text"><h4>MY RECENT WORK</h4></div>
+    <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2">Multi-Post Stories</div>
+      <div class="item3">
+        <div class="bgr"><img src="images/download__1_-removebg-preview.png" alt="Ellipse"></div>
+        <p>
+          A daily selection of privately<br> personalized reads; no accounts<br>
+           or sign-ups required. has been the<br> industry's standard dummy text<br>
+            ever since the 1500s, when an<br> unknown printer took a standard<br> dummy text.
+        </p>  
+      </div>
+      <div>
+        <ul class="htmlcss">
+          <li class="item4">css</li>
+          <li class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item6">html</li>
+          <li class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item8">bootstrap</li>
+          <li class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item10">ruby</li>
+        </ul>
```

Comment:
- [OPTIONAL] As one the requirement is to stick to the Figma design, I can see there are some issues in your design that I will highly recommend to fix them. and the reason of this change is mentioned below.
 - You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
 The issues are the following:
- [OPTIONAL] In your design, the projects' cards' technologies `li`s have bullet points that are the default styling of `li`s in HTML, kindly try to remove them. there are also some mysterious black bullet points in each `li`, I will recommend to fix them as well.
- [OPTIONAL] One of the requirements in the previous milestone is to make our design responsive from 375px up to the 768px. I can see that your design is not responsive as you have given your `body` the exact width of `375px`, it made your design unresponsive. I will highly recommend to fix that. As that problem was in previous pull request `Files changed` tab. now I can't access that and I can't suggest an inline-comment in the specific place, that's why I am writing it here.
your design
![qwer](https://user-images.githubusercontent.com/96683843/212996648-fc1e1f0f-1789-43d7-971e-0969645df91a.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/212996638-cea1aadd-47fb-4f63-86b0-73ff646ae7fb.jpg)